private.go

```diff
@@ -47,6 +55,10 @@ var unroutableCIDR6 = []string{
 }
 
 func init() {
+	privateCIDR4 = append(privateCIDR4, localCIDR4...)
```

Comment:
This is confusing. Could you move the append to the declaration? Also, does it make sense to keep locals here?